packages/api/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -157,6 +165,19 @@ function addSandboxFields(docRef: DocumentReference): DocumentReference {
     // Add a random date to the sandbox document
     if (docRef.content[0]?.attachment) {
       docRef.content[0].attachment.creation = randomDates[randomIndex];
+      docRef.content[0].attachment.size = Math.floor(Math.random() * 100000);
```

Comment:
Can't we use the actual doc's size? Trying to keep it close to real data as possible, as the `sandbox` is how our cxs likely start integrating with our api.

Reply:
Also, now that I see it, shouldn't we keep the original info there if it exists? Both for `creation` and `size`.

Reply:
to do that, we would first need to fetch the documents, which doesn't happen here, but happens later on `GET /document` route. Currently, we already use some hardcoded values for the purposes of displaying an example of what CX could expect to see in prod. Having exact values here is not crucial. But, we could raise a follow up issue for this. For the purposes of this ticket, I think this improvement is already a plus